src/sagas/user.saga.ts

```diff
@@ -12,21 +12,43 @@ function* login(action) {
   const { email, password } = action.payload;
 
   try {
-    //const token =
-    yield userService.login(email, password);
-
+    const resData = yield userService.login(email, password);
+    console.log(resData);
+    const tokenExpirationDate = new Date(new Date().getTime() + 1000 * 60 * 60);
+    localStorage.setItem(
```

Comment:
Imo it will be better to extract anything related to localStorage to a different util file, this way if we want to use some other storage the change will be only in the util file